12_upgrade/60_technique_upgrade.txt

```diff
@@ -4,36 +4,88 @@ At the first installation, Rudder will automatically deploy a Technique library
 `/var/rudder/configuration-repository/techniques` directory.
 
 When upgrading Rudder to another version, a new (updated) Technique library will be deployed
-in `/opt/rudder/share/techniques`, and Rudder will automatically take care of updating the system
+in `/opt/rudder/share/techniques`, and Rudder will only update the *system*
 Techniques in the configuration-repository directory.
 
-However, the other Techniques will not be updated automatically (yet), so you will have to do
-it yourself.
+However, the other Techniques are not updated automatically in Rudder 4.x, so it needs a user action.
```

Comment:
I believe we need to use the same term for the non system techniques coming with Rudder. I propose "Techniques from Rudder standard library"
So "the other Techniques are not" => "Techniques from Rudder standard library are not"